website/content/en/highlights/2021-11-04-remap-dropped-output.md

```diff
@@ -0,0 +1,82 @@
+---
+date: "2021-11-04"
+title: "New `dropped` output on the `remap` transform"
+description: "Sending failed events down a separate pipeline"
+authors: ["lukesteensen"]
+pr_numbers: [9169]
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: "new feature"
+  domains: ["remap"]
+---
+
+Vector 0.18 introduces a new `dropped` output to the `remap` transform. This
+can be used to route events that fail processing down a separate pipeline.
+
+When either of `drop_on_error` or `drop_on_abort` is set to `true`, events that
+are dropped from the primary output stream due to either errors or aborts are
+written instead to a separate output called `dropped`.
```

Comment:
It's worth pointing out that none of the mutations before the error/drop-statement are preserved.

The example below does show that, but it's worth pointing out in a single sentence for people that skip the example.